src/main/java/reborncore/RebornCore.java

```diff
@@ -105,6 +108,16 @@ public void onInitialize() {
 		RebornCoreCommands.setup();
 
 		RebornCoreTags.WATER_EXPLOSION_ITEM.toString();
+		
+		/* register UnloadHandler */
+		if (getSide() == EnvType.CLIENT)
+			ClientBlockEntityEvents.BLOCK_ENTITY_LOAD.register((blockEntity, world) -> {
```

Comment:
Will this crash on a dedicated server? 

I would register the client one in RebornCoreClient and the server one here.

Reply:
I have implemented your suggestion and verified what I have commited now doesn't crash the dedicated server.